packages/framework-integration-tests/integration/helper/app-helper.ts

```diff
@@ -0,0 +1,35 @@
+import { ProviderTestHelper } from '@boostercloud/application-tester'
+import { AWSTestHelper } from '@boostercloud/framework-provider-aws-infrastructure'
+import * as util from 'util'
+
+const exec = util.promisify(require('child_process').exec)
+
+export function applicationName(): string {
+  return `my-store-${process.env.BOOSTER_APP_SUFFIX}`
+}
+
+export async function getProviderTestHelper(): Promise<ProviderTestHelper> {
+  const provider = process.env.TESTED_PROVIDER
+  const providerHelpers: Record<string, () => Promise<ProviderTestHelper>> = {
+    AWS: () => AWSTestHelper.build(applicationName()),
+  }
```

Comment:
Once a provider has implemented the `ProviderTestHelper`, we can add it here

Reply:
🤔  This are the integration test of the framework itself, so we test those providers we officially support. 
The only way to do that is to add the provider to the source code and add a line here.

BUT! the correct® way to do that is: That someone will include the integration tests in their repo, as it is done with the auth package. They could use the new `application-tester` package to help with that